packages/web/app/config/.env.common

```
@@ -35,7 +35,7 @@ NF_GOOGLE_TAG_ID=UA-88781119-2
 # Enables NEUR operations in wallet Purchase and Redeem
 NF_NEURO_WITHDRAW_ENABLED=1
 
-NF_SENTRY_DSN=
+NF_SENTRY_DSN=https://1a01753f602c4bbcad39de77a992fc8d@sdsn.neufund.io/7
```

please remove this we don't need to report the errors to sentry anymore

---

done